docs/src/components/pricing/PricingTable.tsx

```diff
@@ -247,7 +247,7 @@ const Cell = ({ highlighted = false, ...props }: BoxProps & { highlighted?: bool
       flexDirection: 'column',
       alignItems: {
         xs: 'center',
-        md: 'flex-start',
+        md: 'flex-center',
```

Comment:
@mbrookes 

> Edit: Actually, looking at the alignment of the scroll-view header, it seems it might be intentional. Let's revert it and center both. It doesn't look right.

I have iterated both ways, centering and justifying at the start:

| Center      | Start |
| ----------- | ----------- |
| <img width="500" alt="Screen Shot 2021-08-24 at 11 21 40" src="https://user-images.githubusercontent.com/67129314/130635170-0e6ec5a0-bd49-4dd5-a96d-81bf16506fca.png">      | <img width="500" alt="Screen Shot 2021-08-24 at 11 22 02" src="https://user-images.githubusercontent.com/67129314/130635182-a6786bf9-f8b6-42eb-9f96-e5515a031742.png">|

I'm not sure what you mean by _"center both"_ though. Both what? Do you mean the sticky header plans text? 

I'd also like to get more opinions on this as well since it's mostly a matter of "looking right or wrong". To me, personally, justified to the start looks better and right, since it aligns with how every other text on the table is justified, and especially when the sticky header is on view, so the icons match the plan title as you scroll. @siriwatknp and @oliviertassinari Could you share what you think it's best too?

As a quick reference/benchmark: [Mailchimp justifies to the start](https://mailchimp.com/pricing/marketing/compare-plans/) while [Hopin centers the icon](https://hopin.com/pricing).

Reply:
Personally, I prefer `center` (may be make the sticky header center also like `     🔷 Pro.     `). Anyway, I don't see any difference between both in terms of clarity.

Reply:
> Do you mean the sticky header plans text?

Yes, sorry for the ambiguity.